source/exitcodes.rst

```diff
@@ -25,4 +25,8 @@ The 100 exit status code means there are one or more syntax errors in the file b
 
 The 200 exit status code means there are one or more semantic errors in the file being compiled.
 
-<- Insert Example ->
\ No newline at end of file
+.. code :: none
+
+    Errors detected during compilation 
+    Exit code: 200
+    Semantic error at 1:0: Cannot assign expression of type Stack<number> to x, which is of type number
```

Comment:
Could you add a line above showing the code that would produce this error?

Reply:
It's the same as how syntax errors were done so it would be inconsistent unless that is also changed